databricks/assets/dashboards/clusters_dashboard.json

```
@@ -0,0 +1,1997 @@
+                                    },
+                                    "columns": [
+                                        {
+                                            "field": "timestamp",
+                                            "width": "auto"
+                                        },
+                                        {
+                                            "field": "service",
+                                            "width": "auto"
+                                        },
+                                        {
+                                            "field": "resource_name",
+                                            "width": "auto"
+                                        },
+                                        {
+                                            "field": "@state.result_state",
+                                            "width": "auto"
+                                        },
+                                        {
+                                            "field": "@state.life_cycle_state",
+                                            "width": "auto"
+                                        },
+                                        {
+                                            "field": "@duration",
+                                            "width": "auto"
+                                        },
+                                        {
+                                            "field": "@error.message",
+                                            "width": "auto"
+                                        }
+                                    ]
+                                }
+                            ],
+                            "type": "list_stream"
+                        },
+                        "layout": {
+                            "x": 0,
+                            "y": 6,
+                            "width": 12,
+                            "height": 3
+                        }
+                    }
+                ]
+            },
+            "layout": {
+                "x": 0,
+                "y": 34,
+                "width": 12,
+                "height": 10
+            }
+        },
+        {
+            "id": 6145985371833798,
+            "definition": {
+                "title": "Spark Job Performance",
+                "background_color": "blue",
+                "show_title": true,
+                "type": "group",
+                "layout_type": "ordered",
+                "widgets": [
+                    {
+                        "id": 7694220939331754,
+                        "definition": {
+                            "type": "note",
+                            "content": "##  Spark Job Performance\n\nSpark operations and metrics coming from the spark driver. \n\nSpark Jobs and SQL are clickable to display a more detailed view of the computations",
```

I wasn't sure if the clickable element is in Datadog in the tile, if it's outside of Datadog, ignore my suggestion.

```suggestion
                            "content": "##  Spark Job Performance\n\nSpark operations and metrics coming from the spark driver. \n\nClick Spark Jobs and SQL to view a more detailed view of the computations",
```